src/components/picker/picker.tsx

```diff
@@ -344,6 +344,7 @@ export class Picker {
                 visible={!!content}
                 containerId={this.portalId}
                 containerStyle={styling}
+                inheritParentWidth={true}
```

Comment:
Note to self: limel-input-field with autocomplete would need this too, but that doesn't seem to use limel-portal yet, so it's fine for now.